src/Task1.java

```diff
@@ -0,0 +1,84 @@
+package src;
+
+import java.util.*;
+
+public class Task1 {
+    public static int countOccurance(List<String> list, String word) {
+        int count = 0;
+        for (String w : list) {
+            if (w.equals(word)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List toList(int[] arr) {
+        List list = new ArrayList();
```

Comment:
Не используется "Diamond Notation", и так по всему классу.
```suggestion
        List<Integer> list = new ArrayList<>();
```